### ISSUE TITLE
Errno 104 Connection reset by peer

### ISSUE THREAD

**Issue by awesomebytes** [dddomodossola/remi#62]
Its probably just a faulty network killing stuff and there is maybe nothing to do... but I do get:

```
Exception happened during processing of request from ('172.16.15.28', 32928)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 593, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 649, in __init__
    self.handle()
  File "/usr/local/lib/python2.7/dist-packages/remi/server.py", line 164, in handle
    if not self.read_next_message():
  File "/usr/local/lib/python2.7/dist-packages/remi/server.py", line 175, in read_next_message
    length = self.rfile.read(2)
  File "/usr/lib/python2.7/socket.py", line 380, in read
    data = self._sock.recv(left)
error: [Errno 104] Connection reset by peer
```

Connecting a phone to my machine and accessing a non-trivial app.

Is there any try: except: thing we can do?

**Comment by nzjrs** [comment]
When I have seen this it is usually transient and harmless. What fails to work after this error?

**Comment by awesomebytes** [comment]
I see this when pushing a button that updates the contents of a TextInput. Once it happens, that button does not do its full work anymore. If I refresh the page, the new contents are found in the TextInput after pushing the button. But they are not dynamically written anymore. I guess the socket that connects that ability has died and there is no way to recover it :/

**Comment by awesomebytes** [comment]
Well, I can refresh the page and it kinda works.

**Comment by awesomebytes** [comment]
With this PR:
https://github.com/dddomodossola/remi/pull/66

This should also behave better. I think.

**Comment by dddomodossola** [comment]
Should be fixed in this 3e66c90e5f835f2e89202feec7b916acff874e26
